app/controllers/api/reviews_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Api::ReviewsController < ApplicationController
+  respond_to :json
+  skip_before_action :authenticate
+
+  def create
+    review = Review.new(review_params)
+
+    if review.save
+      respond_with review
+    else
+      head :unprocessable_entity
+    end
+  end
+
+  def show
+    review = Review.find(params[:id])
+    respond_with review
```

Comment:
We've had to specify the specific namespace before, perhaps that's your issue. Example:

```ruby
respond_with :api, review
```